packages/frontend/src/pages/ManageEventsPage/components/EventCard/EventCard.tsx

```diff
@@ -31,7 +32,11 @@ export const EventCard = ({ event }: { event: IEvent }) => {
             )}
           >
             <Dot color={event.status === EventStatus.Active ? 'green' : 'gray'} />
-            <div>{translation[(event.status + 'Event') as 'draftEvent']}</div>
+            <div>{event.status === EventStatus.Completed || event.status === EventStatus.Active ?

```

Comment:
typescript type casting doesn't affect runtime code, in runtime it will be 
```
event.status === EventStatus.Completed || event.status === EventStatus.Active ? translation[(event.status + 'Event')] :  translation[(event.status + 'Event')]
```
so this condition doesn't do anything. But for transparency it can be rewritten as 
` <div>{translation[(event.status + 'Event') as 'draftEvent' | 'completedEvent' | 'activeEvent' | 'canceledEvent']}</div>`

Reply:
Done